commands/command.md

```diff
@@ -1,52 +1,80 @@
 Returns @array-reply of details about all Redis commands.
 
-Cluster clients must be aware of key positions in commands so commands can go to matching instances,
-but Redis commands vary between accepting one key,
-multiple keys, or even multiple keys separated by other data.
+Cluster clients must be aware of key positions in commands so commands can go to matching instances, but Redis commands vary between accepting one key, multiple keys, or even multiple keys separated by other data.
+
+Starting from Redis 7.0 a more flexible mechanism was introduced in order to help clients finding the key positions.
+For more information please check the [key-specs page][tr].
+[tr]: /topics/key-specs
 
 You can use `COMMAND` to cache a mapping between commands and key positions for
 each command to enable exact routing of commands to cluster instances.
 
-## Nested Result Array
-Each top-level result contains six nested results.  Each nested result is:
-
-  - command name
-  - command arity specification
-  - nested @array-reply of command flags
-  - position of first key in argument list
-  - position of last key in argument list
-  - step count for locating repeating keys
-
-### Command Name
-
-Command name is the command returned as a lowercase string.
+`COMMAND`'s reply is an @array-reply, where each element is an @array-reply by itself, containing the following elements:
 
-### Command Arity
+ - command name
+ - command arity specification
+ - nested @array-reply of command flags
+ - position of first key in argument list
+ - position of last key in argument list
+ - step count for locating repeating keys
+ - nested @array-reply of ACL categories
+ - nested @map-reply of additional information
 
-<table style="width:50%">
-<tr><td>
-<pre>
-<code>1) 1) "get"
-   2) (integer) 2
-   3) 1) readonly
-   4) (integer) 1
-   5) (integer) 1
-   6) (integer) 1
-</code>
-</pre>
-</td>
-<td>
-<pre>
-<code>1) 1) "mget"
+## Example
+```
+1) 1) "mget"
    2) (integer) -2
    3) 1) readonly
+      2) fast
    4) (integer) 1
    5) (integer) -1
    6) (integer) 1
-</code>
-</pre>
-</td></tr>
-</table>
+   7) 1) @read
+      2) @string
+      3) @fast
+   8)  1) "summary"
+       2) "Get the values of all the given keys"
+       3) "since"
+       4) "1.0.0"
+       5) "group"
+       6) "string"
+       7) "complexity"
+       8) "O(N) where N is the number of keys to retrieve."
+       9) "arguments"
+      10) 1) 1) "name"
+             2) "key"
+             3) "type"
+             4) "key"
+             5) "flags"
+             6) 1) multiple
+             7) "value"
+             8) "key"
```

Comment:
not sure if MGET is a good example, this is a bit complex, maybe find a simpler one)